src/utils/schema-utils.js

```diff
@@ -76,6 +76,11 @@ export function getTypeInfo(schema) {
       constrain += `Max ${constrain ? '┃' : ''}${schema.maxLength} chars`;
     }
   }
+
+  if (dataType.match(/integer|number/g) && schema.format) {
+    info.type = schema.format;
```

Comment:
`type` and `format` are two distinct attributes. we use `type` in many places to make decision on rendering complex schemas. assigning `format` to `type` is technically incorrect and also breaks  many of our rendering logic

Reply:
can you render LONG in SCHEMA correctly and consistently?